gcloud-java-pubsub/baseline/src/main/java/com/google/gcloud/pubsub/spi/v1/SubscriberApi.java

```diff
@@ -66,9 +65,26 @@
  */
 @javax.annotation.Generated("by GAPIC")
 public class SubscriberApi implements AutoCloseable {
+  // ========
```

Comment:
Less likely by the code-generator? Again, feel free to look at other common java libraries (guava, and JDK included).
This is by far not a common practice.